src/pages/Main/Main.js

```diff
@@ -2,46 +2,18 @@ import React, { Component } from "react";
 import { Link } from "react-router-dom";
 import Slider from "react-slick";
 import Answer from "./Answer/Answer";
+import text from "./AnswerData";
 import "./Main.scss";
 
 class Main extends Component {
   state = {
     click: "",
-    text: [
-      {
-        q: "영양제의 가격이 14,000원이 넘지 않아요. 성분이 부족하진 않나요?",
-        answer:
-          "필리 영양제는 한국인 식습관과 각 영양성분의 섭취 용량을 계산해 충분한 요량을 담았어요. 단, 유통마진을 최소화해 거품없는 가겨으로 제공해요.",
-      },
-      {
-        q: "정기구독을 시작하면 약정기간이 있거나 해지가 어렵진 않나요??",
-        answer:
-          "필리 정기구독은 별도의 약정기간이 없어요. 원할때는 홈페이지에 로그인해 언제든 직접 관리할 수 있어요.",
-      },
-      {
-        q: "잊고있다가 자동으로 결제가 되버릴까봐 걱정돼요.",
-        answer:
-          "필리는 정기결제 2일 전에 미리 결제 알림을 카카오 알림톡으로 보내드려요. 가격과 주소 등 확인하고 관리할 수 있으니안심하세요.",
-      },
-      {
-        q: "이번 달 영양제를 다 먹지 못하면 어떡하죠?",
-        answer:
-          "이런 저런 이유로 영양제를 다 드시지 못하셨군요. 걱정하지마세요. 홈페이지에서 정기결제일을 언제든 직접 변경할 수 있어요. 언제든 내맘대로 관리하는 정기구독 시작하세요.",
-      },
-      {
-        q: "매 달 배송료가 발생하나요?",
-        answer:
-          "필리 정기구독을 이용하면 배송비가 무료예요. 배송비는 필리가 책임질게요. 꾸준히 섭취하고 건강을 지키세요.",
-      },
-    ],
   };
 
   answerClick = (index) => {
-    if (this.state.click === index) {
-      this.setState({ click: "" });
-    } else {
-      this.setState({ click: index });
-    }
+    this.state.click === index
+      ? this.setState({ click: "" })
+      : this.setState({ click: index });
```

Comment:
setState 자체가 아니라 객체 안의 value에 삼항연산자를 적용해주면 좋겠습니다.
```suggestion
    this.setState({ click: this.state.click === index ? "" : index })
```

Reply:
넵